addons/pos_event/views/event_event_views.xml

```diff
@@ -5,6 +5,9 @@
         <field name="model">event.event</field>
         <field name="inherit_id" ref="event.view_event_form"/>
         <field name="arch" type="xml">
+            <xpath expr="//div[hasclass('oe_title')]" position="attributes">
+                <attribute name="class" add="mb-4" separator=" "/>
+            </xpath>
```

Comment:
Why is this needed actually?
Is it related to other changes? If not maybe just mention it briefly in the commit message